docs/preprocessing.md

```diff
@@ -0,0 +1,15 @@
+## Executing *fMRIPrep*
+Because *fMRIPrep* creates a single anatomical reference for all sessions, we will need first to run with ``--anat-only`` so that the results get cached and *fMRIPrep* processes do not run into race conditions.
+
+Similarly, when running *fMRIPrep* on a dataset with many sessions, the safest approach is to submit one job per session.
+
+- [ ] Submit the anatomical workflow:
+    ``` bash title="Executing anatomical workflow of fMRIPrep"
+    {% include 'code/fmriprep/ss-fmriprep-anatonly.sh' %}
+    ```
```

Comment:
You need to indent the file for it to render properly:

```suggestion
    ``` bash title="Executing anatomical workflow of fMRIPrep"
    {% filter indent(width=4) %}
    {% include 'code/fmriprep/ss-fmriprep-anatonly.sh' %}
    {% endfilter %}
    ```
```